docs/7_modules.md

```diff
@@ -106,3 +106,446 @@
 ### Build your own form types for modules
 
 > *This is not yet written*
+
+
+### How to create or overwrite a module
+
+A module consists of four files and translations:
+- **config.yaml**: Definition of the module, version, file path for editing and its form, and the fields required by the module.
+- **form.html.twig**: Form for editing, here you can order how you want the fields to be displayed.
+- **edit.html.twig**: Layout for the module edition, which should show how the module will look like. With a series of data attributes you can link the form fields defined in the config.yaml and how they will look like.
+- **render.html.twig**: Final rendering of the module, used in the preview action and in the final rendering of the page.
+
+#### Create
+You can create new modules to suit your project design needs.
+
+In the folder your-project/app/cms/modules, add a folder with the name of the module you want to create, e.g. **rating**.
```

Comment:
/app/ is only for some project structures the regular structure should be **_your-project_/cms/modules**

Reply:
Done!